src/Service/Configurations/RuntimeConfigProvider.cs

```diff
@@ -255,14 +296,6 @@ public virtual bool TryLoadRuntimeConfigValue()
                 throw new ArgumentException($"'{nameof(configuration)}' cannot be null or empty.", nameof(configuration));
             }
 
-            DbConnectionStringBuilder dbConnectionStringBuilder = new()
-            {
```

Comment:
Is this no longer true? SWA may not provide db name in connection string for controller v1?

Reply:
nvm I see now its moved into the function